tests/sals/automated_chatflows/test_solutions.py

```diff
@@ -324,3 +324,29 @@ def test07_4to6gw(self):
         self.info("Check that the ipv6 GW is reachable")
         res, out, err = j.sals.process.execute(f"ping6 -c 1 google.com")
         self.assertFalse(res)
+
+    def test08_etcd(self):
+        """Test case for deploying etcd.
+
+        **Test Scenario**
+
+        - Deploy etcd.
+        - Check that etcd is reachable.
+        - Check put a Hello by etcdctl.
+        """
+        self.info("Deploy etcd")
+        name = self.random_name()
+        etcd = deployer.deploy_etcd(solution_name=name, pool=self.pool_id, network=self.network_name,)
+        self.solution_uuid = etcd.solution_id
+
+        self.info("Check that etcd is reachable")
+        self.assertTrue(
+            self.wait(etcd.ip_addresses[0], port=2379, timeout=self.timeout),
```

Comment:
I think the ip is a private wg ip. how do we reach the etcd with ip ? shouldn't it be with domain ?

Reply:
it's in the setup method